build.sbt

```diff
@@ -22,7 +22,7 @@ libraryDependencies ++= Seq(
   "org.apache.spark" %% "spark-core" % sparkVersion,
   "org.apache.spark" %% "spark-sql" % sparkVersion,
   "org.apache.spark" %% "spark-streaming" % sparkVersion
-).map(_ % "provided")
+)
```

Comment:
I'm not sure about this, but should we leave the `"provided"` part in this so as to not include these in our fat jar?